templates/hotgraphic.hbs

```diff
@@ -17,7 +17,13 @@
         {{/if}}
         {{#each _items}}
           <button class="base hotgraphic-graphic-pin component-item-text-color {{#if _isVisited}}visited{{/if}}" data-index="{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" aria-label="{{title}}.{{#if _isVisited}} {{../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
-            <div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}"></div>
+            {{#if this._pin.src}}
```

Comment:
sorry I know this is quite picky - but there's no need to use `this` here and - to the best of my knowledge - we don't ever include it in any other .hbs files in core adapt. would you mind removing it for consistency?

Reply:
No need to apologise! I'd rather you be picky